README.md

```diff
@@ -11,6 +11,9 @@ This is the Orders business logic module.
 
 ## Additional information
 
+IMPORTANT NOTE! Only in case an acquisition unit has to be assigned to the Order it is required that user should have 
+extra permission `orders-storage.acquisitions-unit-assignments.item.post` to create an purchase order.
```

Comment:
please change the permission name to "orders.assignments.item.post"

Reply:
Just realized we both posted the same comments at the same time, Going to remove the duplicates